subworkflows/local/longread_hostremoval.nf

```diff
@@ -20,13 +20,13 @@ workflow LONGREAD_HOSTREMOVAL {
     ch_multiqc_files  = Channel.empty()
 
     if ( !params.longread_hostremoval_index ) {
-        ch_minimap2_index = MINIMAP2_INDEX ( [ [], reference ] ).index.map { it[1] }
+        ch_minimap2_index = MINIMAP2_INDEX ( [ [], reference ] ).index
```

Comment:
Why was this removed?

Reply:
Due to the the input reference channel change in the process `MINIMAP2_ALIGN`. The new module needs meta for index: `tuple val(meta2), path(reference)`